src/services/neo4jService.js

```diff
@@ -181,5 +194,40 @@ export default {
         .catch((error) => {
           return {status: "ERROR", result: error};
         });
+        
   },
+  getRelationshipData: async function (driver, query, params) {
+    const session = driver.session();
+    return await session
```

Comment:
i'd like to rewrite this and all existing functions without using `.then` and `.catch`
but again this would cause a lot of unrelated changes

Reply:
i mean just async/await syntax